Xamarin.Forms.Core/RadioButton.cs

```diff
@@ -37,11 +38,12 @@ public string GroupName
 			set { SetValue(GroupNameProperty, value); }
 		}
 
-		public ImageSource ButtonSource
-		{
-			get { return (ImageSource)GetValue(ButtonSourceProperty); }
-			set { SetValue(ButtonSourceProperty, value); }
-		}
+		// TODO Needs implementations beyond Android
+		//public ImageSource ButtonSource
```

Comment:
what happens right now if people use ImmageSource?

Reply:
It’s kind of a funny API anyway. You’re able to set a custom image for the checkbox, but you need it together with the VSM to dynamically set the checked state. See the example in the gallery I commented out. 

If people use it on platforms where it’s not implemented, nothing happens right now.